junction/proposals/services.py

```diff
@@ -30,3 +32,23 @@ def comment_recipients(proposal_comment):
             .all().select_related('commenter')}
     recipients.add(proposal.author)
     return recipients
+
+
+def send_mail_for_new_proposal(conference, title, author, speaker_info,
+                               url, proposal_section_id, host):
+    proposal_section = ProposalType.objects.get(pk=proposal_section_id)
+    send_to = [c.reviewer for c in
+               ConferenceProposalReviewer.objects.filter(conference=conference)]
+    for to in send_to:
+        if to == author:
+            continue
+        send_email(to=to,
+                   template_dir='proposals/email/proposal',
+                   context={'to': to,
+                            'title': title,
```

Comment:
You could pass the proposal object directly rather than passing individual values into context. So this would become `context={'proposal': proposal, ...}`.